app/models/work.rb

```diff
@@ -31,6 +31,10 @@ class Work < ApplicationRecord
       transition first_draft: :deposited, version_draft: :deposited
     end
 
+    event :submit_for_review do
+      transition first_draft: :pending_approval
```

Comment:
And we'll add `version_draft: :pending_approval` later when we work on versioning, I assume

Reply:
yes.